tests/test_crud_immunisation_api.py

```diff
@@ -61,15 +61,15 @@ def test_crud_immunization_nhs_login(nhsd_apim_proxy_url, nhsd_apim_auth_headers
     # UPDATE
     update_payload = copy.deepcopy(imms)
     update_payload["id"] = imms_id
-    update_payload["status"] = "not-done"
+    update_payload["status"] = "entered-in-error"
```

Comment:
Should the data for this one be changed to use some valid 'not-done' data rather than use a different status? Not sure 'entered-in-error' is a sensible status for an update as wouldn't erroneous data be logically deleted and a new create done for the correct data?

Reply:
It's irrelevant because it's just testing whether doing an update and retrieving the data works. It doesn't matter what it's updating, just that it is updating successfully. I'm guessing status was chosen because it's an easy field to access.

Reply:
Ok, in that case we can leave that as is